ens-names/proto/v1/ens.proto

```diff
@@ -18,4 +18,8 @@ message Domain {
 
 message Domains {
     repeated Domain items = 1; 
-}
\ No newline at end of file
+}
+
+message Lines {
```

Comment:
why not use `sf.substreams.sink.files.v1` instead?

Reply:
The `codegen.rs` file removes all the generated files starting with `sf.substreams` name. One way to solve this was to edit the `codegen.rs` file not remove the generated file and the second approach was to create my own proto similar to `sf.substreams.sink.files.v1`. I thought the second approach was cleaner.